### DISCUSSION TITLE
Add lazyloading

### DISCUSSION THREAD

**Discussion by aerni** [spatie/statamic-responsive-images#198] [Ideas]
What are your thoughts about adding simple lazyloading with `loading="lazy"`? It's a simple thing to improve performance. This should optionally be disableable on a tag level, because it's considered bad practice to lazyload any images above the fold.

**Comment by ncla** [comment]
Yes, this is good idea. I assume this is not neatly achievable already by publishing the Blade template in it's current version.

**Reply by aerni** [reply]
I did publish the view to add the loading attribute. That's no problem. But this will add lazyloading to all images, which is not always desirable. So it would be nice if we could disable lazyloading from the tag.

**Reply by ncla** [reply]
As a workaround for now you can use the available `$attributeString` variable in .blade.php template.

In Antlers:

```antlers
{{ responsive:responsive_field lazyLoad="true" }}
```

In Blade (stripped down template to demonstrate):

```twig
    <img
        src="{{ $src }}"

        @if(\Illuminate\Support\Str::contains($attributeString, 'lazyLoad="1"'))
            loading="lazy"
        @endif
    >
```

**Reply by aerni** [reply]
Oh sweet! I didn't know this worked. Thanks!
